elixir/apps/web/lib/web/mailer/auth_email.ex

```diff
@@ -59,4 +59,24 @@ defmodule Web.Mailer.AuthEmail do
       remote_ip: "#{:inet.ntoa(remote_ip)}"
     )
   end
+
+  def new_user_email(
+        %Domain.Accounts.Account{} = account,
+        %Domain.Actors.Actor{} = actor,
```

Comment:
Is this variable used? I checked the template and can't find it there

Reply:
Ah, yeah, you're right.  I added this by mistake.  I'll take it out.  Thanks for catching this!